userguide/jails.rst

```diff
@@ -198,6 +198,11 @@ Setting a proxy in the %brand%
 to use the proxy settings, except when performing DNS lookups. Make sure
 a firewall is properly configured to maximize system security.
 
+When pairing the jail with a physical interface, edit the
+:ref:`interface <Interfaces>` and set
+:guilabel:`Disable Hardware Offloading`. This prevents a network
+interface reboot when the jail is powered on.
```

Comment:
Much better, but "powered on" is an unclear concept for a jail.  Does this mean when the jail starts or any time the jail is running?  Probably when it starts:
```suggestion
interface reset when the jail starts.
```